tests/acceptance/features/webUIFavouritesActionMenu/favouritesActionMenu.feature

```diff
@@ -0,0 +1,16 @@
+Feature: Visibilty of the file or folder actions menu options on favourites folder
+    As a user
+    I want to see the actions menu options of the file or folder on favourites folder
+    So that the menu options of the selected file type or folder are visible to me
+
```

Comment:
we put 2 empty lines before each scenarios. like you did in `tests/acceptance/features/webUIBatchAction/batchAction.feature`

Reply:
Note: we do that so that we can tag scenarios in future, and still have an empty line between the previous scenario and the tags of the next scenario. And we can do that without line-numbers changing (that might be referenced in expected-failures files or...)

Reply:
I have done it in recent commit.